lib/gemsurance/gem_info_retriever.rb

```diff
@@ -45,17 +46,45 @@ def ==(other)
           @status == other.instance_variable_get(:@status) &&
           @vulnerabilities == other.vulnerabilities
       end
+
+      def self.attributes
+        ['name', 'current_version', 'newest_version', 'in_gem_file', 'homepage_uri', 'documentation_uri', 'source_code_uri', 'status', 'vulnerabilities']
+      end
+
+      def attributes
+        self.class.attributes
+      end
+
+      def values
+        self.instance_variables.map{|ivar| self.instance_variable_get ivar}
+      end
+
+      def to_csv
+        formatted_values.to_csv
+      end
+
+      private
+
+      def formatted_values
+        self.instance_variables.map do |ivar|
```

Comment:
Same comment here. It would be great if we could do this without reference the `instance_variables` array or using `instance_variable_get`.